docs/src/main/asciidoc/resource-handling.adoc

```diff
@@ -48,19 +48,19 @@ NOTE: If you choose to use a security principal to authenticate and authorize wi
 
 |*spring.cloud.azure.storage.blob*.enabled
 |true
-|Whether an Azure Blob Storage Service is enabled
+|Whether to enable Azure Storage Blob.
```

Comment:
> Whether to enable Azure Storage Blob.
I saw you add "." at the end of the line,
do we need to add "." for each entry?

Reply:
We'd better do that for configuration properties. check this https://docs.spring.io/spring-boot/docs/current/reference/html/application-properties.html